pkg/proxy/metrics/metrics.go

```diff
@@ -138,6 +138,17 @@ var (
 		},
 	)
 
+	// IptablesCTInvalidDroppedPackets is the number of packets dropped
+	// by iptables which were marked INVALID by conntrack.
+	IptablesCTInvalidDroppedPackets = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Subsystem:      kubeProxySubsystem,
+			Name:           "iptables_ct_invalid_dropped_pkt",
+			Help:           "packets dropped by iptables which were marked invalid by conntrack",
```

Comment:
That makes it sound like "iptables dropped this packet and then conntrack marked it invalid" which is backwards.

"packets dropped by iptables because conntrack marked them invalid"

or

"packets dropped by iptables to work around conntrack problems"